steps:
  - name: pypyr.steps.pype
    in:
      pype:
        name: 36_own_failure_handler
        failure: arb_group
